components/NewReplySection/ReplyForm/ReferenceInput.js

```diff
@@ -13,7 +13,6 @@ const useStyles = makeStyles(theme => ({
     },
   },
   textarea: {
-    flex: 1,
```

Comment:
Thank you for locating the root cause!

After trying the UI out, I found that `flex: 1` is actually required on mobile to enlarge the textarea.

I suggest leaving `flex: 1` as-is and adding the following in `[theme.breakpoints.up('md')]`: 
```
      flex: 'none', // Disable flex so that user can enlarge textarea by themselves
      minHeight: 144
```

Reply:
Thanks for your review. 
I had applied the suggestions.